src-java/kilda-pce/src/main/java/org/openkilda/pce/AvailableNetworkFactory.java

```diff
@@ -102,22 +102,18 @@ public AvailableNetwork getAvailableNetwork(Flow flow, Collection<PathId> reuseP
         return network;
     }
 
-    private boolean isPrimaryPath(Flow flow, FlowPath flowPath) {
-        return flowPath.getPathId().equals(flow.getForwardPathId())
-                || flowPath.getPathId().equals(flow.getReversePathId());
-    }
-
-    private Collection<Isl> getAvailableIsls(BuildStrategy buildStrategy, Flow flow) {
+    private Collection<IslImmutableView> getAvailableIsls(BuildStrategy buildStrategy, Flow flow) {
         if (buildStrategy == BuildStrategy.COST) {
-            Collection<Isl> isls = flow.isIgnoreBandwidth()
-                    ? islRepository.findAllActiveByEncapsulationType(flow.getEncapsulationType())
-                    : islRepository.findActiveWithAvailableBandwidth(flow.getBandwidth(), flow.getEncapsulationType());
+            Collection<IslImmutableView> isls = flow.isIgnoreBandwidth()
+                    ? islRepository.findActiveByEncapsulationType(flow.getEncapsulationType())
+                    : islRepository.findActiveByBandwidthAndEncapsulationType(flow.getBandwidth(),
+                    flow.getEncapsulationType());
```

Comment:
Confusing line alignment, I have failed to seatch where this argument is passed for first / second / third and several more glances.

Reply:
Agree. I've rewritten with if/else instead of the ternary operator.